### DISCUSSION TITLE
backup constantly loses the same file

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#220] [Q&A]
My first attempt to backup an SD card from my SONY camera,

`root@mk-lbb:/media/storage/202307071648-3887/private/M4ROOT/CLIP# ls -la
total 5880000
drwxrwxrwx 3 www-data www-data      32768 Jul 10 17:07 .
drwxrwxrwx 6 www-data www-data      32768 Dec 10  2022 ..
-rwxrwxrwx 1 www-data www-data 2546991104 Jul  8 17:13 .C0003.MP4.PYPJ3R
-rwxrwxrwx 1 www-data www-data 1313603584 Jul 10 17:08 .C0003.MP4.ZfEPS1
-rwxrwxrwx 1 www-data www-data  280950644 Dec 10  2022 C0001.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0001M01.XML
-rwxrwxrwx 1 www-data www-data 1879396697 Dec 10  2022 C0002.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0002M01.XML
drwxrwxrwx 2 www-data www-data      32768 Jul  7 22:07 tims
root@mk-lbb:/media/storage/202307071648-3887/private/M4ROOT/CLIP# ls -la
total 5894336
drwxrwxrwx 3 www-data www-data      32768 Jul 10 17:07 .
drwxrwxrwx 6 www-data www-data      32768 Dec 10  2022 ..
-rwxrwxrwx 1 www-data www-data 2546991104 Jul  8 17:13 .C0003.MP4.PYPJ3R
-rwxrwxrwx 1 www-data www-data 1328283648 Jul 10 17:08 .C0003.MP4.ZfEPS1
-rwxrwxrwx 1 www-data www-data  280950644 Dec 10  2022 C0001.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0001M01.XML
-rwxrwxrwx 1 www-data www-data 1879396697 Dec 10  2022 C0002.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0002M01.XML
drwxrwxrwx 2 www-data www-data      32768 Jul  7 22:07 tims
root@mk-lbb:/media/storage/202307071648-3887/private/M4ROOT/CLIP# ls -la
total 5909440
drwxrwxrwx 3 www-data www-data      32768 Jul 10 17:07 .
drwxrwxrwx 6 www-data www-data      32768 Dec 10  2022 ..
-rwxrwxrwx 1 www-data www-data 2546991104 Jul  8 17:13 .C0003.MP4.PYPJ3R
-rwxrwxrwx 1 www-data www-data 1343750144 Jul 10 17:08 .C0003.MP4.ZfEPS1
-rwxrwxrwx 1 www-data www-data  280950644 Dec 10  2022 C0001.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0001M01.XML
-rwxrwxrwx 1 www-data www-data 1879396697 Dec 10  2022 C0002.MP4
-rwxrwxrwx 1 www-data www-data       2003 Dec 10  2022 C0002M01.XML
drwxrwxrwx 2 www-data www-data      32768 Jul  7 22:07 tims`

so that video C0003.MP4. starts getting copied, then fails, and gets restafrted over and over again

The main log on the LBB menu keeos stuck at 27 or 1105 files (or such) and does not move

Where is that log kept in the first place (so i can search it from an SSH session)?

I find no indication that there is any read failure or such (nothing in dmesg(1) output) but after a while
when the copy had gotten to 8M copied, it starts over again

Then after a longer while /media.source just vanishes and never gets remounted

I am also confused with the main menu's [Mount USB1 ]  [Remove USB2]
menu points

it does not state which is which
and at times it shows 'mount uSB2]. when in fact both /media/source and /media/storage *are* mounted,

**Comment by outdoorbits** [comment]
Could you try again using a powered hub? I had problems like that in backups from smartphones as camera too

**Comment by sejtam** [comment]
This is using the Orico powered hub. I managed to find one locally.
I am powering the RPi using a 3A PSU and the hub a 65w adapter
(though my cable shows it only darwing 3w itself)

If there were power issues, I'd expect some messages in dmesg(1) output
regarding power issues?

**Comment by outdoorbits** [comment]
Could you solve the problem?
